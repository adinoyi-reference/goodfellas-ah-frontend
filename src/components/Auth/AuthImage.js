import React from 'react';
import author from '../../assets/author.jpg';

const AuthImage = () => (
  <div className="author-writing">
    <p className="auth-header">Authors Haven</p>
    <img width="84%" src={author} alt="" />
    <p className="auth-title">A community of like minded authors</p>
  </div>
);

export default AuthImage;
