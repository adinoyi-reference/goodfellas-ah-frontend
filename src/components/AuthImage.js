import React from 'react';
import author from '../assets/author.jpg';

const AutnImage = () => (
  <div className="authorWriting">
    <p className="authHeader">Authors Haven</p>
    <img width="84%" src={author} alt="" />
    <p className="authTitle">A community of like minded authors</p>
  </div>
);

export default AutnImage;
