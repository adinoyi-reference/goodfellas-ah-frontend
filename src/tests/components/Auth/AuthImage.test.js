import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AuthImage from '../../../components/Auth/AuthImage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<AuthImage />);
});

describe('AuthImage', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
