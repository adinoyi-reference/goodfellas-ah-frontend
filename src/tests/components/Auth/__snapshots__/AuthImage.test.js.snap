// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthImage render features view should render as expected 1`] = `
<div
  className="author-writing"
>
  <p
    className="auth-header"
  >
    Authors Haven
  </p>
  <img
    alt=""
    src="https://res.cloudinary.com/drmmqcxkc/image/upload/v1541518106/author.jpg"
  />
  <p
    className="auth-title"
  >
    A community of like minded authors
  </p>
</div>
`;
